frontend/src/app/applications/permit-application-list/permit-application-list.component.html

```diff
@@ -0,0 +1,85 @@
+<section class="usa-section">
+  <div class="usa-grid">
+    <h1>Mount Baker-Snoqualmie National Forest</h1>
+    <div class="application-status-toggle-wrapper">
+      <label for="application-status-toggle">Filter by</label>
+      <select class="usa-input-inline" id="application-status-toggle" (change)="applicationStatusChange($event)">
+        <option value="pending" selected>Pending</option>
+        <option value="accepted">Accepted</option>
+        <option value="rejected">Rejected</option>
+        <option value="cancelled">Cancelled</option>
+        <option value="expired">Expired</option>
+      </select>
+    </div>
+    <h2><span class="permit-application-type">{{applicationStatus}}</span> permit applications</h2>
+    <div *ngIf="apiErrors" class="usa-alert usa-alert-error" role="alert">
```

Comment:
Seems like having a widely used alert component would also make sense so that if we ever wanted to tweak the alerts that could be done centrally throughout the application.

Reply:
I'll create an issue for this